src/sfconfig.js

```diff
@@ -1,6 +1,6 @@
 export default {
   api: {
-    server: 'http://172.32.1.144:8000/api'
+    server: 'http://localhost:8000/api'
```

Comment:
Maybe we should get this file out of git repo...

Reply:
it's up to you